src/components/StatsLeaderboard.vue

```diff
@@ -19,11 +19,10 @@
         multiple
       />
     </div>
-    <!-- https://vuetifyjs.com/en/api/v-data-table/ -->
-    <v-data-table
+    <!-- https://next.vuetifyjs.com/en/api/v-table/ -->
+    <v-table
```

Comment:
Literally none of these props will work, there's a whole different setup to do a normal table vs. data table. This is just temporary to fix the linting error.